automation/index.html

```diff
@@ -112,14 +120,28 @@ <h1>Deploy the Web Vitals GTM Template with GA4</h1>
         </label>
       </li>
       </ul>
+      <div class="library-decision">
+        <label id="use-unpkg-check">
+        Use the defaut library from unpkg?
+        <input type="checkbox" checked name="use-unpkg" id="use-unpkg" class="use-unpkg">
+      </label>
+      <label class="library-url-field">
+        URL to fetch the web-vitals.js library from:
+        <input type="text" name="library-url" id="library-url">
+      </label>
+      </div>
       <label>
         Use attribution build of the web-vitals library?
         <input type="checkbox" checked name="use-attribution" id="use-attribution">
       </label>
```

Comment:
In the template I only display this when the user selected the "Load library from unpkg.com" option, because the attribution build URL is known then. If the user selects to load the web vitals library from a custom URL I hide this option, because we can only rely on the provided URL (and I assume the user has chosen the correct build then).

I recommend hiding this label and input field with a condition based on chosen option of class "library-decision".

Reply:
Ok, so based on the attribution flag set by the user you are setting event parameters in the GA4 tag for the attribution data, right? Then it makes sense to leave it as is.

Reply:
Thanks. I'll get the last bits in and then we can iterate on the completed PR for main.